tests/acceptance/pageObjects/phoenixPage.js

```diff
@@ -231,6 +247,14 @@ module.exports = {
     acceptSharesInNotifications: {
       selector: '//div[@id="oc-notification"]//h5[contains(text(),\'%s\')]/../div/button/span[.="Accept"]',
       locateStrategy: 'xpath'
+    },
+    userProfile: {
+      selector: '//div/button[@id="_userMenuButton"]',
+      locateStrategy: 'xpath'
+    },
+    profileInfoContainer: {
+      selector: '//div[@class="uk-card-body uk-flex uk-flex-middle uk-flex-column"]',
```

Comment:
Please see if there's any better selectors.

Reply:
no there is not